recipes/megahit/build.sh

```diff
@@ -3,6 +3,10 @@
 #strictly use anaconda build environment
 CC=${PREFIX}/bin/gcc
 CXX=${PREFIX}/bin/g++
+export CFLAGS="-I$PREFIX/include"
+export LDFLAGS="-L$PREFIX/lib"
+export C_INCLUDE_PATH=${PREFIX}/include
+export LIBRARY_PATH=${PREFIX}/lib
```

Comment:
You could try 
```
export CPATH=${PREFIX}/include
export CPP_INCLUDE_PATH=${PREFIX}/include
```
if this also does not work you could try to patch this line:
`
CXXFLAGS = -g -O2 -Wall -Wno-unused-function -Wno-array-bounds -D__STDC_FORMAT_MACROS -funroll-loops -fprefetch-loop-arrays -fopenmp -I. -std=c++0x -static-libgcc $(CPU_ARCH)
`

in the Makefile to include `-I$PREFIX/include`